boards/arduino_mkrwifi1010/README.md

```diff
@@ -0,0 +1,27 @@
+# Arduino MKR WiFi 1010 Board Support Crate
+
+This crate provides a type-safe API for working with the [Arduino MKR WiFi 1010 board](https://store.arduino.cc/usa/mkr-wifi-1010).
+
+## Examples
```

Comment:
This is straight from the Arduino Nano 33 IoT crate, I don't know if this works yet. Will update this once I've actually tested it on the real board.